content/source/docs/cloud/api/changelog.html.md

```diff
@@ -0,0 +1,51 @@
+---
+page_title: "API Changelog - API Docs - Terraform Cloud and Terraform Enterprise"
+layout: "cloud"
+page_id: "api-changelog"
+---
+
+[breaking]: ./api/changelog/breaking.png "Breaking"
+[cloud]: ./api/changelog/cloud.png "Cloud"
+[enterprise]: ./api/changelog/enterprise.png "Enterprise"
+
+# API Changelog
+
+Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
+
+### 2021-05-27
+
+* ![cloud][] [Agents](https://www.terraform.io/docs/cloud/api/agents.html): added [delete endpoint](https://www.terraform.io/docs/cloud/api/agents.html#delete-an-agent).
+
+### 2021-05-19
+
+* [Runs](https://www.terraform.io/docs/cloud/api/runs.html): added `refresh`, `refresh-only`, and `replace-addrs` attributes.
+
+### 2021-04-16
+
+* Introduced [Controlled Remote State Access](https://www.hashicorp.com/blog/announcing-controlled-remote-state-access-for-terraform-cloud-and-enterprise).
+    * [Admin Settings](https://www.terraform.io/docs/cloud/api/admin/settings.html): added `default-remote-state-access` attribute.
+    * [Workspaces](https://www.terraform.io/docs/cloud/api/workspaces.html): added `global-remote-state` attribute.
+    * [Workspaces](https://www.terraform.io/docs/cloud/api/workspaces.html): added [Remote State Consumers](https://www.terraform.io/docs/cloud/api/workspaces.html#get-remote-state-consumers) relationship.
+
+### 2021-04-13
+
+* [Teams](https://www.terraform.io/docs/cloud/api/teams.html): added `manage-policy-overrides` property to the `organization-access` attribute.
+
+### 2021-03-23
+
+* ![enterprise][] Introduced [Share Modules Across Organizations with Terraform Enterprise](https://www.hashicorp.com/blog/share-modules-across-organizations-terraform-enterprise).
```

Comment:
@bethanyr @jbonhag I amended this to follow suit for the others, but am missing details for the actual changes to each API. Do you mind filling in the `...` for it?

Reply:
I have updated this section:
![image](https://user-images.githubusercontent.com/4753589/120729242-7a897f00-c493-11eb-9aa8-057f1b47f903.png)